rust/datafusion/src/execution/physical_plan/parquet.rs

```diff
@@ -110,6 +112,12 @@ struct ParquetPartition {
     iterator: Arc<Mutex<dyn RecordBatchReader + Send + Sync>>,
 }
 
+impl Debug for ParquetPartition {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        f.debug_struct("ParquetPartition").finish()
```

Comment:
As above, this is not `#derive` because `RecordBatchReader` does not implement `Debug`